src/runtimes/docker/utils/spawnStreamAsync.ts

```diff
@@ -195,6 +195,8 @@ export async function spawnStreamAsync(
             disposable.dispose();
             if (code === 0) {
                 resolve();
+            } else if (signal) {
+                reject(new ChildProcessError(`Process exited due to signal ${signal}`, code, signal));
```

Comment:
This change should be made upstream in https://github.com/microsoft/vscode-docker-extensibility as well

Reply:
OK I will submit a separate PR for that